File: ardoise-project/alpha-0/project/sketch.js
var bg;
var pizza;

function setup() {
	bg = loadImage("img/blackboard.jpg");
	createCanvas(windowWidth, windowHeight);	
}

function draw() {
	background(bg);
	fill(255);
	if (mouseIsPressed) {
		ellipse(mouseX, mouseY, 100, 100);
	}
}

File: hello-processing/app/src/main/java/com/example/stephane/hello_processing/Sketch.java
package com.example.stephane.hello_processing;

/**
 * Created by stephane on 8/11/2017.
 */

import processing.core.PApplet;

public class Sketch extends PApplet {
    public void settings() {
        size(600, 600);
    }

    public void setup() {
        fullScreen();
        background(0);
    }

    public void draw() {
        fill(255);
        if (mousePressed) {
            ellipse(mouseX, mouseY, 50, 50);
        }
    }
}